### DISCUSSION TITLE
How to configure monorepo?

### DISCUSSION THREAD

**Discussion by MichaelKapustey** [pnpm/pnpm#4015] [Q&A] [answered]
I've expected to have a predefined command (`pnpm workspace init`?) that would do everything for monorepo like `npm init` does for single package. However, seems like even documentation on how to do this by hands is missing or deeply hidden somewhere. Could you guide me?

**Comment by zkochan** [comment, marked as answer]
You only need to create a [pnpm-workspace.yaml](https://pnpm.io/pnpm-workspace_yaml) in the root of the monorepo.

**Comment by MichaelKapustey** [comment]
What about content? Include everything
```
packages:
  - './**'
```

**Reply by zkochan** [reply]
```yaml
packages:
  - "**"
```
